src/fortune/service.py

```diff
@@ -0,0 +1,84 @@
+# src/fortune/service.py
+from datetime import date
+from typing import Optional, List
+from fastapi import Depends, HTTPException, status
+from sqlalchemy.ext.asyncio import AsyncSession
+from src.fortune.entities.models import DailyFortuneResource as DailyFortuneResourceModel
+from src.fortune.entities.schemas import UserDailyFortuneSummary
+from src.fortune.entities.schemas import (
+    DailyFortuneResource,
+    DailyFortuneResourceCreate,
+    DailyFortuneResourceUpdate,
+    DailyFortuneResourceList,
+)
+from src.fortune.repository import FortuneRepository
+from src.common.dependencies import get_db_session
+from src.fortune.entities.enums import FortuneType
+
+
+class FortuneService:
+    def __init__(
+        self,
+        fortune_repository: FortuneRepository = Depends(),
+        session: AsyncSession = Depends(get_db_session),
+    ):
+        self.repository = fortune_repository
+        self.session = session
+
+    async def list_fortunes(
+        self,
+        cursor: Optional[int],
+        limit: int,
+        publish_date: Optional[date],
+        fortune_type: Optional[FortuneType],
+    ) -> DailyFortuneResourceList:
+        items, next_cursor = await self.repository.get_fortunes(
+            cursor=cursor, limit=limit, publish_date=publish_date, fortune_type=fortune_type
+        )
+        return DailyFortuneResourceList(
+            items=[DailyFortuneResource.model_validate(i) for i in items],
+            next_cursor=next_cursor,
+        )
+
+    async def create_fortune(self, dto: DailyFortuneResourceCreate) -> DailyFortuneResource:
+        model = DailyFortuneResourceModel(
+            publish_date=dto.publish_date,
+            fortune_type=dto.fortune_type,
+            image_url=dto.image_url,
+            description=dto.description,
+        )
+        self.session.add(model)
+        await self.session.commit()
```

Comment:
commit은 middleware에서 요청 완료되면 한번에 하고 있어서, repository에서 commit으로 작성한 부분들은 일괄로 flush로 바꿔도 될 것 같아~~
```suggestion
        await self.session.flush()
```